src/PoolManager.sol

```diff
@@ -407,19 +368,15 @@ contract PoolManager is Auth, IPoolManager {
             undeployedTranche.decimals,
             trancheTokenWards
         );
-        address restrictionManager = restrictionManagerFactory.newRestrictionManager(
-            undeployedTranche.restrictionSet, token, restrictionManagerWards
-        );
-
-        TrancheTokenLike(token).file("restrictionManager", restrictionManager);
+        ITrancheToken(token).file("hook", undeployedTranche.hook);
```

Comment:
I think we need to require that the hook address is set

Reply:
What do you mean?

Reply:
when you add the tranche, the hook can be set to address(0)
https://github.com/centrifuge/liquidity-pools/blob/c6dce19190bb7dc32372830a0be410b4ba6c7d3c/src/PoolManager.sol#L245

Should we allow deployments without a valid hook address?

Reply:
The comment in line 244 answers that :)